mackerel-plugin-linux/lib/linux_test.go

```diff
@@ -167,3 +142,28 @@ pswpout 113`
 	assert.EqualValues(t, stat["pswpin"], 0)
 	assert.EqualValues(t, stat["pswpout"], 113)
 }
+
+func TestCollectDiskstats(t *testing.T) {
+	path := "/sys"
+
+	_, err := os.Stat(path)
+	if err != nil {
+		return
+	}
+	p := make(map[string]interface{})
+
+	assert.Nil(t, collectDiskStats(path, &p))
```

Comment:
Does it work correctly when running this test on non-linux OS?

Reply:
On non-linux OS, there is no '/sys' so it will be returned early.

Reply:
ok